lib/matplotlib/tests/test_ft2font.py

```diff
@@ -823,7 +824,7 @@ def test_ft2font_drawing():
     np.testing.assert_array_equal(image, expected)
     font = ft2font.FT2Font(file, hinting_factor=1, _kerning_factor=0)
     glyph = font.load_char(ord('M'))
-    image = ft2font.FT2Image(expected.shape[1], expected.shape[0])
+    image = np.zeros(expected.shape, np.uint8)
```

Comment:
@anntzer maybe my previous comment was not as clear because I'm not into the details. It was motivated by changes like this where the order of dimensions is reversed. I found this noteworthy.

Reply:
Sure, done.